test/extended/images/helper.go

```diff
@@ -16,3 +24,48 @@ func GetImageLabels(c client.ImageStreamImageInterface, imageRepoName, imageRef
 	}
 	return image.Image.DockerImageMetadata.Config.Labels, nil
 }
+
+// ModifySourceCode will modify source code in the pod of the application
+// according to the sed script.
+func ModifySourceCode(oc *exutil.CLI, selector labels.Selector, sedScript, file string) error {
```

Comment:
this feels oddly scoped to me.  why not have this method take a pod or a container, instead of a label, given that it only works if the thing it finds is a single pod and it only actualy operates on the first container in that pod?